templates/plugin/remote_execution_ssh.yml.erb

```diff
@@ -4,6 +4,9 @@
 :local_working_dir: <%= scope.lookupvar('::foreman_proxy::plugin::remote_execution::ssh::local_working_dir') %>
 :remote_working_dir: <%= scope.lookupvar('::foreman_proxy::plugin::remote_execution::ssh::remote_working_dir') %>
 :kerberos_auth: <%= scope.lookupvar('::foreman_proxy::plugin::remote_execution::ssh::ssh_kerberos_auth') %>
+<% ssh_log_level = scope.lookupvar('::foreman_proxy::plugin::remote_execution::ssh::ssh_log_level'); if ssh_log_level -%>
+:ssh_log_level: <%= ssh_log_level %>
+<% end -%>
```

Comment:
Curious your thoughts on the `if` inclusion here versus setting a default value to the default from the plugin (`:error`) ?

Reply:
I have a mild preference for unset parameters to not appear in the config file.  At the time, I was planning to rework the gem to allow the `ssh_config(5)` values to be passed directly through.  The gem doesn't allow `DEBUG3` to be set and there is a check that prevents  `:ssh_log_level: debug` from being used unless the entire foreman-proxy is also set to `DEBUG`. Now that it sounds like serious work in that gem is imminent I'm not sure it is worth the effort.

Reply:
> I have a mild preference for unset parameters to not appear in the config file

Sorry, what I was getting at is why not set it to `:error` rather than `undef`.

Reply:
That is the question I answered.  The gem already has a hard coded default value for when the configuration isn't present in the foreman-proxy plugin config file.